src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -306,9 +306,12 @@ private BracketChecker(String field) {
     private static class YearChecker implements Checker {
 
         private static final Predicate<String> CONTAINS_FOUR_DIGIT = Pattern.compile("([^0-9]|^)[0-9]{4}([^0-9]|$)").asPredicate();
+        private static final Predicate<String> ENDS_WITH_FOUR_DIGIT = Pattern.compile("[0-9]{4}$")
+                .asPredicate();
+        private static final String regex = "[](){},.;!?<>%&$]";
 
         /**
-         * Checks, if the number String contains a four digit year
+         * Checks, if the number String contains a four digit year and ends with it
```

Comment:
Please add a comment pointing to the BiBTeX manual where this decision to accept around 1997 and so on is explained.